lib/src/base-styles/_nx-tables.scss

```diff
@@ -126,6 +126,10 @@
   }
 }
 
+.nx-table-row--filter-header > .nx-cell--header {
+  top: $nx-table-celle-header-height;
```

Comment:
This variable should also be used where the height of the header cells is defined, otherwise there's no point in it being a variable.  That would be at https://github.com/sonatype/sonatype-react-shared-components/pull/165/files#diff-9f8d48f03d669e6a348fce7c92efdc55b39676f5a11135268ee172296d57fb6fR157

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/ee72b488d3e7a87119e0736c14c050e0b39c3250